runtime/jvmti/jvmtiThreadGroup.c

```diff
@@ -180,23 +186,36 @@ jvmtiGetThreadGroupChildren(jvmtiEnv* env,
 			/* Construct the Children Groups array under a lock */
 
  			threadGroupObject = *((j9object_t*) group);
+#if defined(J9VM_OPT_OJDK_THREAD_SUPPORT)
+			threadGroupObject = (j9object_t) currentThread->javaVM->internalVMFunctions->objectMonitorEnter(currentThread, threadGroupObject);
+			if (threadGroupObject == NULL) {
```

Comment:
Constant on the left.

Reply:
This is replaced w/ `J9_OBJECT_MONITOR_ENTER_FAILED()`.